src/Features/Core/Portable/CodeFixes/CodeFixService.cs

```diff
@@ -411,6 +437,13 @@ public CodeFixProvider GetSuppressionFixer(string language, ImmutableArray<Diagn
             return false;
         }
 
+        private bool IsInteractiveCodeFixProvider(CodeFixProvider provider)
+        {
+            // TODO (https://github.com/dotnet/roslyn/issues/4932): Don't restrict CodeFixes in Interactive
+            return provider is AddImport.AbstractAddImportCodeFixProvider ||
```

Comment:
@jmarolf As we discussed offline, I think our goal is more nuanced than that.  We will probably also want to be able to invoke code fixes in read-only buffers and accept that the portions of those fixes that applied to the initiating buffer will have no effect (e.g. Extract Method can be useful even if the call site can't be updated).